content/master/concepts/compositions.md

```diff
@@ -1130,6 +1132,100 @@ spec:
         - type: None
 ```
 
+#### Match a Condition
+{{<hover label="condition" line="11">}}Condition{{</hover>}} considers the composed
+resource to be ready when it finds the expected condition type, with the
+expected status for it in its `status.conditions`.
+
+For example, consider 
+{{<hover label="condition" line="7">}}my-resource{{</hover>}}, which will be marked
+as ready if there is a condition of type 
+{{<hover label="condition" line="13">}}MyType{{</hover>}} with a status of
+{{<hover label="condition" line="14">}}Success{{</hover>}}.
+
+```yaml {label="condition",copy-lines="none"}
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+# Removed for Brevity
+spec:
+  resources:
+  # Removed for Brevity
+    - name: my-resource
+      base:
+        # Removed for brevity
+      readinessChecks:
+        - type: MatchCondition
+          matchCondition:
+            type: MyType
+            status: Success
+```
+
+#### Match a Boolean
+
+Two types of checks exist for matching boolean fields:
+ * `MatchTrue`
+ * `MatchFalse`
+
+`MatchTrue` considers the composed resource to be ready when the value of a 
+field within that resource is `true`.
+
+`MatchFalse` considers the composed resource to be ready when the value of a 
+field within that resource is `false`.
+
+For example, consider 
+{{<hover label="matchTrue" line="7">}}my-resource{{</hover>}}, which will be marked
+as ready if 
+{{<hover label="matchTrue" line="12">}} status.atProvider.manifest.status.ready{{</hover>}}
+is {{<hover label="matchTrue" line="11">}}true{{</hover>}}.
+
+```yaml {label="matchTrue",copy-lines="none"}
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+# Removed for Brevity
+spec:
+  resources:
+  # Removed for Brevity
+    - name: my-resource
+      base:
+        # Removed for brevity
+      readinessChecks:
+        - type: MatchTrue
+          fieldPath: status.atProvider.manifest.status.ready
+```
+{{<hint "tip" >}}
+Checking {{<hover label="matchTrue" line="11">}}MatchTrue{{</hover>}} doesn't
+require a `match` field.
+{{< /hint >}} 
```

Comment:
Do you mean the `matchCondition:` field?

Reply:
No, it's more like a `matchSomethingSomething` field. Eg, the `MatchInteger` type has a companion `mathInteger` type.

`MatchTrue` and `MatchFalse` don't require additional fields. Which I guess would be a better way to describe this:

```suggestion
{{<hint "tip" >}}
Checking {{<hover label="matchTrue" line="11">}}MatchTrue{{</hover>}} doesn't
require setting additional fields.
{{< /hint >}} 
```

Sounds good?

PS: I copied this section from [here](https://github.com/crossplane/docs/blob/969b94d2809ae3d826c9c02ed890925e8c0d14be/content/master/concepts/compositions.md?plain=1#L1077), I'm guessing it would also make sense to adjust that. Wdyt?

https://github.com/crossplane/docs/blob/969b94d2809ae3d826c9c02ed890925e8c0d14be/content/master/concepts/compositions.md?plain=1#L1106-L1109

Reply:
Ah, apparently "additional" is a bit too wordy for Vale. :thinking: 

This seems ok:
```suggestion
{{<hint "tip" >}}
Checking {{<hover label="matchTrue" line="11">}}MatchTrue{{</hover>}} doesn't
require setting any other fields.
{{< /hint >}} 
```

Reply:
@plumbis just pushed a commit with this fix, lmkwyt :grinning: